internal/storage/storage.go

```diff
@@ -40,13 +60,25 @@ func (s *storage) Set(key string, value any) error {
 	if callback != nil {
 		callback(key, value)
 	}
+
+	err := s.jsonStorage.Set(key, value)
+	if err != nil {
+		return err
+	}
 	return nil
 }
 
 func NewStorage(opts ...storageOption) StorageWithCallbacks {
+	// we ignore the error and just work without locking
+	file, err := xdg.ConfigFile("snyk/ls-config")
+	if err != nil {
```

Comment:
How will this be ignored if it will panic? Or are you relying on the recovery mechanism?